xcode/Subconscious/Shared/Services/Noosphere/NoosphereService.swift

```diff
@@ -140,6 +140,12 @@ final class NoosphereService: SphereProtocol, SphereIdentityProtocol {
         guard let identity = self._sphereIdentity else {
             throw NoosphereServiceError.defaultSphereNotFound
         }
+        
+        return try open(identity: identity)
+    }
+    
+    /// Open a sphere using its identity
+    func open(identity: String) throws -> Sphere {
```

Comment:
Unsure if this actually makes sense as an API. I want to be able to manage multiple `Sphere`s so this seems like a logical inclusion.